### ISSUE TITLE
thank you for your work！Can you share  ‘DDAM_msceleb.pt‘？

### ISSUE THREAD

**Issue by Stellafx** [simon20010923/DDAMFN#1]

**Comment by simon20010923** [comment]
Thank you for your interest in our project. I'm on a business trip now. I will pose the pretrain model after 01/01/2024. I will cue you when it's available. Please **STAR** our project and listen to our calls.

**Comment by simon20010923** [comment]
The code is updated and the pretrained model is available.

**Comment by simon20010923** [comment]
Our new version DDAMFN++ is available in the folder https://github.com/simon20010923/DDAMFN/DDAMFN++, which includes new alignment code refer to https://github.com/biubug6/Pytorch_Retinaface and optimized by SAM ([Sharpness aware minimization for efficiently improving generalization](https://arxiv.org/abs/2010.01412)). The trained RAF-DB model is available in DDAMFN++/checkpoints_ver2.0, which is not our best. The affectnet model will soon be available.

**Comment by simon20010923** [comment]
Our new version DDAMFN++ is available in the folder https://github.com/simon20010923/DDAMFN/tree/main/DDAMFN%2B%2B, which includes new alignment code refer to https://github.com/biubug6/Pytorch_Retinaface and optimized by SAM ([Sharpness aware minimization for efficiently improving generalization](https://arxiv.org/abs/2010.01412)). The trained RAF-DB model is available in DDAMFN++/checkpoints_ver2.0, which is not our best. The affectnet model will soon be available.

**Comment by Stellafx** [comment]
Congratulations on your good results！
 Sorry to bother you again, but when you conducted the experiment on ferplus, did you use the official validation set or the test set for your code's test set?

**Comment by Stellafx** [comment]
Congratulations on your good results！
 Sorry to bother you again, but when you conducted the experiment on ferplus, did you use the official validation set or the test set for your code's test set?
